protostar/testing/fuzzing/strategy_inference.py

```diff
@@ -11,6 +15,8 @@ def infer_strategy_from_cairo_type(
 ) -> StrategyDescriptor:
     if isinstance(cairo_type, TypeFelt):
         return FeltsStrategyDescriptor()
+    if isinstance(cairo_type, TypeStruct) and str(cairo_type.scope).endswith("Uint256"):
```

Comment:
Can identification be performed in some other way? Like the structure itself, not the name?

Reply:
@Arcticae See this comment thread: https://github.com/software-mansion/protostar/pull/977#discussion_r995783576